python/vowpalwabbit/DFtoVW.py

```diff
@@ -738,19 +876,21 @@ def check_label_type(self):
         TypeError
             If label is not of type SimpleLabel or MulticlassLabel.
         """
-        available_labels = (SimpleLabel, MulticlassLabel, MultiLabel)
+        available_labels = (
+            SimpleLabel,
+            MulticlassLabel,
+            MultiLabel,
+            ContextualbanditLabel,
+        )
 
-        if self.label is None:
-            pass
-        else:
-            if not isinstance(self.label, available_labels):
-                raise TypeError(
-                    "Argument 'label' should be either of the following type: {label_types}.".format(
-                        label_types=repr(
-                            [x.__name__ for x in available_labels]
-                        )[1:-1]
-                    )
+        label = self.label[0] if isinstance(self.label, _ListLabel) else self.label
+
+        if not isinstance(label, available_labels):
```

Comment:
When self.label is a _ListLabel, the underlying type checked is the one of the _instances_ in the list (by default the first one).